src/bci_build/package.py

```diff
@@ -2016,6 +2048,50 @@ def _get_os_container_package_names(os_version: OsVersion) -> tuple[str, ...]:
     for os_version in ALL_BASE_OS_VERSIONS
 ]
 
+# https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp3991.pdf
+# Chapter 9.1 Crypto Officer Guidance
+_FIPS_15_SP2_ASSET_BASEURL = "https://api.opensuse.org/public/build/"
+_FIPS_15_SP2_BINARIES = [
+    f"SUSE:SLE-15-SP2:Update/pool/x86_64/openssl-1_1.18804/{name}-1.1.1d-11.20.1.x86_64.rpm"
+    for name in ("openssl-1_1", "libopenssl1_1", "libopenssl1_1-hmac")
+] + [
+    f"SUSE:SLE-15-SP1:Update/pool/x86_64/libgcrypt.15117/{name}-1.8.2-8.36.1.x86_64.rpm"
+    for name in ("libgcrypt20", "libgcrypt20-hmac")
+]
+
+FIPS_BASE_CONTAINERS = [
+    OsContainer(
+        name="base-fips",
+        package_name="base-fips-image",
+        exclusive_arch=[Arch.X86_64],
```

Comment:
Why x86_64 only?

Reply:
because the rest of the packages are x86_64 specific

Reply:
> the original request is rancher which only exists for that architecture.

ok